dist/index.html

```diff
@@ -126,28 +124,29 @@ <h2>Total Cost</h2>
     </section>
     <section class="pantry hidden" id="pantry-view">
       <section class="pantryContents">
-        <div class="dropDropMenu">
+        <menu class="dropDropMenu">
           <label for="ingredients">Add Ingredient to Pantry: </label>
             <select name="ingredients" id="ingredient-drop-down-menu" class="selectMenu">
             </select>
             <form class="quantity">
               <input type="text" placeholder="quantity" class="quantityInput" id="quantity-input">
             </form>
             <button type="button" class="addButton" id="add-button">Add to Pantry</button>
-        </div>
-        <div class="pantryItems">
-            <div class="ingredientsPantry">
-            <div class="pantryHeading">
+        </menu>
+        <section class="pantryItems">
+          <section class="ingredientsPantry">
+            <article class="pantryHeading">
               <h3>Ingredients</h3>
               <h3>Amount</h3>
-            </div>
-            <div class="table" id="pantry-table">
-              <div class="boxI"></div>
-              <div class="boxA"></div>
+            </article>
+            <datalist class="table" id="pantry-table">
```

Comment:
@knpausch, I am learning SO much about semantic HTML tags reviewing your code!! Thank you!!!